Documentation/specs/runtime-configuration-file.md

```diff
@@ -338,9 +236,9 @@ The shared framework's deps file will also contain a `runtimes` section defining
 
 The host will have a RID embedded in it during compilation (for example, `win10-x64` for Windows 64-bit). It will look up the corresponding entry in the `runtimes` section to identify what the fallback list is for `win10-x64`. The fallbacks are identified from most-specific to least-specific. In the case of `win10-x64` and the example above, the fallback list is: `"win10-x64", "win10", "win81-x64", "win81", "win8-x64", "win8", "win7-x64", "win7", "win-x64", "win", "any", "base"` (note that an exact match on the RID itself is the first preference, followed by the first item in the fallback list, then the next item, and so on).
```

Comment:
`The host will have a RID embedded in it during compilation`
This line is incorrect. Can you fix it?  We do RID detection in the host now.

Reply:
Chaning to "The host will detect the RID at runtime"